### PR TITLE
custom num strategy, faster test

### LINKED ISSUE #1486: proptest strategy with better extreme edge cases
The default strategy in proptest does not cover well extreme edge cases, e.g.

` proptest::collection::vec(any::<u64>(), 1..5))`

does very seldom yield something like: 

`
vec![1572656989877777, 1170935903116329, 720575940379279, 0]
`

We should create a custom `Strategy`, to more often pass extreme values, to cover edge cases
```
    fn num_strategy() -> impl Strategy<Value = u64> {
        prop_oneof![
            (u64::MAX-10..64::MAX).prop_map(|num| num as u64),
            (0..10u64).prop_map(|num| num as u64),
            (0..1000u64).prop_map(|_| any::<u64>() )),
        ]
    }
```

### PR DESCRIPTION
**Pull request by PSeitz**
closes #1486
faster test with rand values

### CODE REVIEW

**Review by fulmicoton — APPROVED** [review]

File: fastfield_codecs/src/lib.rs [hunk 1]
```diff
@@ -153,21 +154,32 @@ mod tests {
...
+        fn test_proptest_large(data in proptest::collection::vec(num_strategy(), 1..6000)) {
             create_and_validate::<LinearCodec>(&data, "proptest linearinterpol");
             create_and_validate::<BlockwiseLinearCodec>(&data, "proptest multilinearinterpol");
             create_and_validate::<BitpackedCodec>(&data, "proptest bitpacked");
         }
 
     }
+    fn num_strategy() -> impl Strategy<Value = u64> {
+        prop_oneof![
+            1 => prop::num::u64::ANY.prop_map(|num| u64::MAX - (num % 10) ),
```
Comment (fulmicoton) [inline fastfield_codecs/src/lib.rs:178]: that's awesome!
